privacy/scrubber.go

```diff
@@ -87,11 +88,8 @@ func (scrubber) ScrubRequest(bidRequest *openrtb2.BidRequest, enforcement Enforc
 	var userExtParsed map[string]json.RawMessage
 	userExtModified := false
 
-	var userCopy *openrtb2.User
-	userCopy = ptrutil.Clone(bidRequest.User)
-
-	var deviceCopy *openrtb2.Device
-	deviceCopy = ptrutil.Clone(bidRequest.Device)
+	userCopy := ptrutil.Clone(bidRequest.User)
+	deviceCopy := ptrutil.Clone(bidRequest.Device)
```

Comment:
VS Code "yelled" at me to make this nitpick optimization.

Reply:
Oh so does it actually work? I also had an error that looked like it's not compiling, so I added those types explicitly.

Reply:
![Screenshot 2023-09-05 at 5 04 15 PM](https://github.com/prebid/prebid-server/assets/4790765/e03aa278-61a9-4a51-98ee-6bff6d4b30b3)

Reply:
> Maybe we should put it back?

Auto-completion works fine for me on VS Code with Go 1.20.5. Is there an update available for IntelliJ?

Reply:
Just updated. It didn't help. 
This is just one line of code that doesn't take memory and CPU, but makes IDEs happy. 
> Maybe we should put it back? 

![Screenshot 2023-09-06 at 11 53 51 AM](https://github.com/prebid/prebid-server/assets/4790765/60f2d547-dc45-4ed1-9862-b52953961114)

Reply:
Let's make a deal. I'll revert and could you please open a bug report for IntelliJ?